sycl/include/sycl/ext/oneapi/properties/property.hpp

```diff
@@ -169,6 +188,12 @@ template <typename PropertyT> struct IsRuntimeProperty : std::false_type {};
 // Trait for identifying compile-time properties.
 template <typename PropertyT> struct IsCompileTimeProperty : std::false_type {};
 
+// Trait for property compile-time meta names and values.
+template <typename PropertyT> struct PropertyMetaInfo {
+  static constexpr const char *name = "";
+  static constexpr std::nullptr_t value = nullptr;
```

Comment:
Shouldn't it be empty here to cause compile time errors when used with unknown `PropertyT` ?

Reply:
We don't expect all properties to have meta-information, as not all of them will be relevant to the compiler. For example, properties with runtime values are unlikely to have any relevant information at compile-time, so by default they will just be ignored, which is the behavior of having an empty meta-name.